.dev/docker/env/Dockerfile

```
@@ -36,8 +35,32 @@ RUN apt-get update \
       libvtk${VTK_VERSION}-dev \
       libvtk${VTK_VERSION}-qt-dev \
       qtbase5-dev \
+      software-properties-common \
  && rm -rf /var/lib/apt/lists/*
 
+# Eigen patch (https://eigen.tuxfamily.org/bz/show_bug.cgi?id=1462) to fix issue metioned
+# in https://github.com/PointCloudLibrary/pcl/issues/3729 is available in Eigen 3.3.7
+# Not needed from 20.04 since it is the default version from apt
+RUN if [ `apt show libeigen3-dev 2>/dev/null | grep '^version' -i | awk -F'[ -.]' '{print $4}'` -lt 7 ]; then \
+ && wget -qO- https://gitlab.com/libeigen/eigen/-/archive/3.3.7/eigen-3.3.7.tar.gz | tar xz \
```

```suggestion
  wget -qO- https://gitlab.com/libeigen/eigen/-/archive/3.3.7/eigen-3.3.7.tar.gz | tar xz \
```

Also, the whitespace is not uniform